src/sandstorm/web-session.capnp

```diff
@@ -76,7 +76,17 @@ interface WebSession @0xa50711a14d35a8ce extends(Grain.UiSession) {
   # `clientStream` is the capability which will receive server -> client messages, while
   # serverStream represents client -> server.
 
-  # TODO(someday): Add WebDAV and CalDAV methods?
+  # WebDAV methods
+  propfind @7 (path :Text, content :PropfindContent, context :Context) -> Response;
+  proppatch @8 (path :Text, content :ProppatchContent, context :Context) -> Response;
+  mkcol @9 (path :Text, content :MkcolContent, context :Context) -> Response;
+  copy @10 (path :Text, context :Context) -> Response;
+  move @11 (path :Text, context :Context) -> Response;
+  lock @12 (path :Text, content :LockContent, context :Context) -> Response;
+  unlock @13 (path :Text, content :UnlockContent, context :Context) -> Response;
+  acl @14 (path :Text, content :AclContent, context :Context) -> Response;
+  report @15 (path :Text, content :ReportContent, context :Context) -> Response;
+  options @16 (path :Text, context :Context) -> Response;
```

Comment:
I think it would be bad form to include things in the interface that won't be honored if we can avoid it.

It might make sense to have an "options()" method that doesn't operate like the other methods, but instead returns a "struct Options" which initially will only have a boolean "isDav" field. That way if there are other options we want to support in the future, we can explicitly add them rather than add new methods.